contributors/design-proposals/public-identities-for-pods.md

```diff
@@ -0,0 +1,132 @@
+# Public identities for Pods
+
+* Authors: Sebastian Łaskawiec (@slaskawi)
+* Last edit: [2017-03-13](#history)
+* Status: design
+
+Approvers:
+* [ ] smarterclayton
+* [ ] bprashanth
+* [ ] enisoc
+* [ ] foxish
+* [ ] janetkuo
+* [ ] kargakis
+* [ ] kow3ns
+
+**Table of Contents**
+
+* [Goals](#goals)
+* [Non-goals](#non-goals)
+* [API](#api)
+* [Behavior](#behavior)
+* [Upgrading](#upgrading)
+* [Implementation](#implementation)
+* [Alternatives](#alternatives)
+* [History](#history)
+
+# Goals
+
+* The main goal of Public Identities for Pods is to expose each Pod as an external IP address.
+  This approach allows sophisticated applications to communicate with the outside world using custom protocols.
+  A examples which require this approach are:
+  * Game Servers (which often use RTP, UDP)
+  * Data Grids (with L7 routing, such as [Infinispan](http://infinispan.org/docs/dev/user_guide/user_guide.html#using_hot_rod_server))
+  * Media Servers
+
+* The secondary goal is to create an API similar to `LoadBalancer` to manage external IP addresses.
```

Comment:
Yes and no. I wanted to propose a similar API to Load Balancers and give cloud vendors a chance to allocate an IP address available externally. I imagine some of them (cloud vendors) might not want to abuse Load Balancers (they might be expensive and @smarterclayton mentioned that checking their health might be tricky).